### PR TITLE
Added json file with name

### PR DESCRIPTION
**Pull request by kanishbodhwani**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1781"></a>

### CODE REVIEW

File: public/data/kanishbodhwani.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      "url": "https://kanishportfolio.netlify.app/",
+      "icon": "link"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/kanishbodhwani",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
Comment (Panquesito7) [inline public/data/kanishbodhwani.json:18]: I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

Comment (kanishbodhwani) [reply public/data/kanishbodhwani.json:18]: Yeah LinkedIn is better. Changing it.

**Review by Panquesito7 — APPROVED** [review]
Looks great. Thanks for contributing! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 👍

**Pull request merged**